docker/run-sdo-owner-services.sh

```diff
@@ -23,7 +23,10 @@ Additional environment variables (that do not usually need to be set):
   SDO_RV_PORT - port number RV should listen on *inside* the container. Default is 8040.
   SDO_OPS_PORT - port number OPS should listen on *inside* the container. Default is 8042.
   SDO_OPS_EXTERNAL_PORT - external port number that RV should tell the device to reach OPS at. Defaults to the internal OPS port number.
-  SDO_OCS_API_PORT - port number OCS-API should listen on *inside* the container. Default is 9008.
+  SDO_OCS_API_PORT - port number OCS-API should listen on for HTTP. Default is 9008.
+  SDO_OCS_API_TLS_PORT - port number OCS-API should listen on for TLS. Default is the value of SDO_OCS_API_PORT. (OCS API does not support TLS and non-TLS simultaneously.) Note: you can not set this to 9009, because OCS listens on that port internally.
```

Comment:
Should we also mention `8040` and `8042` ... those are the `ops` and `rv` service ports (or just generally that it should not conflict with any ports defined or modified 'below')

Reply:
Well, that is true for all of the `*_PORT` env vars. I only specifically mentioned 9009, because the knee-jerk reaction would be to just increment from `SDO_OCS_API_PORT`.

Also, note that in the readme, the use of `run-sdo-owner-services.sh` is only documented in the [For Developers Only](https://github.com/open-horizon/SDO-support/blob/master/README-1.10.md#for-developers-only) section. That's because usually `sdo-owner-services` is started by IEAM or all-in-1. The only people that normally have to use `run-sdo-owner-services.sh` are lorenzo and i, so i'm not too concerned about making the documentation dummy-proof. :)